modules/apps/frontend-icons/frontend-icons-web/src/main/resources/META-INF/resources/js/pages/instance_settings/AddIconPackModal.tsx

```diff
@@ -68,7 +68,17 @@ export default function AddIconPackModal({
 		},
 	});
 
+	const nameTaken = !existingIconPackName && !!icons[iconPackName];
+
+	const containsWhiteSpace = !!iconPackName.match(/\s/);
```

Comment:
I think that the empty line between these 2 declarations is unnecessary, as we don't use `nameTaken` on line 73. But it's whatever 🤷